components/multiselect/tag-mode.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Tag Mode
+page_title: MultiSelect Tag Mode
+description: Explore the available tag modes in the Blazor MultiSelect.
```

Comment:
Starting with an action verb is great, just try adding more content specific to this article. Aim at around 150 symbols.